tools/site_and_regional/default_data.cfg

```diff
@@ -0,0 +1,28 @@
+[main]
+clmforcingindir = /glade/p/cesmdata/inputdata
+
+[datm_gswp3]
+dir = /glade/p/cgd/tss/CTSM_datm_forcing_data/atm_forcing.datm7.GSWP3.0.5d.v1.c170516
+domain = domain.lnd.360x720_gswp3.0v1.c170606.nc
+solardir = Solar
+precdir = Precip
+tpqwdir = TPHWL
+solartag = clmforc.GSWP3.c2011.0.5x0.5.Solr.
+prectag = clmforc.GSWP3.c2011.0.5x0.5.Prec.
+tpqwtag = clmforc.GSWP3.c2011.0.5x0.5.TPQWL.
+solarname = CLMGSWP3v1.Solar
+precname = CLMGSWP3v1.Precip
+tpqwname = CLMGSWP3v1.TPQW
+
+[surfdat]
+dir = lnd/clm2/surfdata_map/release-clm5.0.18
+surfdat_16pft = surfdata_0.9x1.25_hist_16pfts_Irrig_CMIP6_simyr2000_c190214.nc
+surfdat_78pft = surfdata_0.9x1.25_hist_78pfts_CMIP6_simyr2000_c190214.nc
+
+[landuse]
+dir = lnd/clm2/surfdata_map/release-clm5.0.18
+landuse_16pft = landuse.timeseries_0.9x1.25_hist_16pfts_Irrig_CMIP6_simyr1850-2015_c190214.nc
+landuse_78pft = landuse.timeseries_0.9x1.25_hist_78pfts_CMIP6_simyr1850-2015_c190214.nc
+
+[domain]
+file = share/domains/domain.lnd.fv0.9x1.25_gx1v7.151020.nc
```

Comment:
any data needed for creating mesh files here?

Reply:
Yes once we add those methods we will (may? depending on how we create the mesh) need to update this file to include the mesh file locations.

Reply:
You only need mesh files for regional cases. And I'm not sure if we've got that all worked out yet. I think that should be a future improvement.

Reply:
This will be handled in #1513 and #1397. So I would put this as a future development.